tests/test_scalers.py

```diff
@@ -10,21 +10,30 @@
 @given(value=floats(-1, 1), threshold=floats(0, 1, exclude_max=True))
 def test_deadzone(value, threshold):
     result = apply_deadzone(value, threshold)
-    assert abs(result) <= 1
-    if value == 0:
+    if value in (-1, 0, 1):
+        assert result == value
+    elif math.isclose(value, 0, abs_tol=threshold):
         assert result == 0
-    elif not math.isclose(value, 0, abs_tol=threshold):
-        assert math.copysign(1, result) == math.copysign(1, value)
+    elif threshold == 0:
+        assert result == value
+    elif value > 0:
+        assert 0 < result < value
+    else:
+        assert value < result < 0
 
 
-@given(value=floats(-1, 1), base=floats(1, exclude_min=True, allow_infinity=False))
+@given(
+    value=floats(-1, 1),
+    base=floats(1, exclude_min=True, allow_infinity=False, width=16),
```

Comment:
Restricting the required precision here to avoid compounding floating point error for floats close to 1. Given that we will usually use "round" numbers for the exponential base, I think this is reasonable.